gen/aircraft.py

```diff
@@ -724,7 +726,40 @@ def generate_planned_flight(
                     start_type=flight.start_type,
                     at=carrier_group,
                 )
+                return group
             else:
+                # If the flight is an helicopter flight, then prioritize dedicated helipads
+                if flight.unit_type.helicopter:
+                    helipad = cp.get_free_helipad()
+                    if helipad is not None and helipad.static_unit is not None:
+                        group = self._generate_at_group(
+                            name=name,
+                            side=country,
+                            unit_type=flight.unit_type.dcs_unit_type,
+                            count=flight.count,
+                            start_type=flight.start_type,
+                            at=helipad.static_unit,
+                        )
+
+                        # Note : A bit dirty, need better support in pydcs
+                        group.points[0].action = PointAction.FromGroundArea
+                        group.points[0].type = "TakeOffGround"
+                        if flight.start_type != "Cold":
+                            group.points[0].action = PointAction.FromGroundAreaHot
+                            group.points[0].type = "TakeOffGroundHot"
+
+                        helipad.occupied = True
+                        group_generated = True
```

Comment:
Unused

Reply:
Resolved in 9ea1edf9dbc8b21c7204ca6a71eab847ddc63522